src/index.rst

```diff
@@ -489,12 +496,12 @@ Workflow cancel/retry
 
 To abort the entire workflow the following script could be used:
```

Comment:
script -> command?

Also at the moment, "abort the entire workflow" is more like abort all non-running jobs for a workflow (running jobs will continue running).   Please either reword this statement or add a separate note.

What one should see on the monitoring pages (e.g., workflow goes through states...)?

Reply:
Also there should be a note about being able to cancel other users' workflows as that would probably be unexpected since authentication is required for the cancel.